source/_components/switch.mqtt.markdown

```diff
@@ -103,6 +103,35 @@ value_template:
   description: "Defines a [template](/docs/configuration/templating/#processing-incoming-data) to extract a value from the payload."
   required: false
   type: string
+device:
+  description: 'Information about the device this switch is a part of to tie it into the [device registry](https://developers.home-assistant.io/docs/en/device_registry_index.html). Only works through [MQTT discovery](/docs/mqtt/discovery/) and when [`unique_id`](#unique_id) is set.'
```

Comment:
`Information about the device this switch` sounds/reads weird.
Adding comma's would probably help.